src/components/store/createStore.ts

```diff
@@ -0,0 +1,63 @@
+import { EditorState } from '../../../types/store/editorState';
+import { Reducer } from '../../../types/store/reducer';
+import { Action } from '../../../types/store/action';
+import { Store } from '../../../types/store/store';
+import * as _ from '../utils';
+
+/**
+ * This function is an entry point to use the store in the editor
+ * It creates the store with an initial state
+ *
+ * It returns functions to use the store:
+ *  subscribe - function for subscribing to each state changes
+ *  dispatch - function for applying actions to the store
+ *  getState - function returns a current state of the store
+ *
+ * @param reducer - current Editor reducer function
+ * @param initialState - initial state of the store
+ */
+function createStore(reducer: Reducer, initialState: EditorState = { blocks: {} }): Store {
```

Comment:
please, describe in jsdoc why do we pass the reducer to the Store. Is there are some usecases when there will be a different reducer? Or it is just a DI for the tests

Reply:
It's not a part of the Store. I can pass different reducers to the Store for different tasks. Also, I can implement function to combine reducers and pass it to the Store (we don't need it right now, but it will help us to store not only blocks data).

Reply:
Please, describe it in a jsdoc, not in a PR.

> for different tasks

Some examples will be helpful. I still can't get the idea.

Reply:
We have the reducer for blocks (I can rename `reducer` function to `blocksReducer`):
- This reducer works with `state.blocks` field;
- Has logic only for blocks actions.

If I want to work with Editor parameters, for example with Editor version:
- I will create `parametersReducer`;
- This reducer will work with `state.parameters.version` field;
- It has logic only for Editor parameters.

So, I have two different reducers and I need to use this two reducers together. I can create `combineReducers` function and combine all reducers to one big reducer and pass it to `createStore` function.

I need to store new data -> I create a new reducer -> I add the reducer to other reducers -> I don't need to change the logic of `createStore` function

I will add information about it to JSdoc. The main idea is that I can add new data to the storage without changing the storage creation interface.